tests/test_merge_lora.py

```diff
@@ -33,3 +35,37 @@ def test_merge_lora(tmp_path, fake_checkpoint_dir):
     keys = base_model.load_state_dict(merged, strict=True)
     assert not keys.missing_keys
     assert not keys.unexpected_keys
+
+
+@RunIf(min_cuda_gpus=1)
+# platform dependent cuda issue: libbitsandbytes_cpu.so: undefined symbol: cquantize_blockwise_fp16_nf4
+@pytest.mark.xfail(raises=AttributeError, strict=False)
```

Comment:
Is this happening only in the GPU CI? Locally it is fine?

Reply:
Locally I don't have a GPU, only CPU and BNB doesn't support anything other than CUDA.
Tried in Studio without this xfail statement and all went fine. Didn't tested in CI without this check, but I assume everything should be fine too.
I assume this `platform dependent` is for Windows. But in order to use BNB in Windows you need to use WSL2, so you'll use Linux anyway, since BNB is shipped with binaries that are compiled for Linux.

It's better to ask @carmocca since I believe it was initially his code that everyone reuses in tests.